src/test/java/bio/terra/integration/IngestTest.java

```diff
@@ -133,7 +133,9 @@ public void ingestBuildSnapshot() throws Exception {
         createdSnapshotIds.add(snapshotSummary.getId());
     }
 
+    // TODO: Revisit this test when permissions work is done!
```

Comment:
Can you make a JIRA for this instead of (or in addition to) this TODO?

Reply:
Yes, definitely. I will before merging.

Reply:
Actually, with @nmalfroy 's input, we were able to go ahead and fix the test! But, I will make tickets to track these changes.